CONTRIBUTING.md

```diff
@@ -144,7 +144,10 @@ Make these #defines with a name that more clearly states what it's for.
 ###Control statements:
 (if,while,for,etc)
 
+* All control statements must have a space before the expression (eg: `if (blah)` not `if(blah)`)
```

Comment:
I mean, heres the thing, if we ARE going to enforce a requirement, I personally think the version i like should be the one we enforce

:^)